iModelCore/ECDb/ECDb/ECSchemas/ECDbMap.02.00.02.ecschema.xml

```diff
@@ -51,4 +51,12 @@
         <ECProperty propertyName="AllowDuplicateRelationships" typeName="boolean" description="Default: false. If set to true duplicate relationships are allowed."/>
     </ECCustomAttributeClass>
 
+    <ECCustomAttributeClass typeName="ImportRequiresVersion" modifier="Sealed" appliesTo="Schema" description="Causes a schema to only be imported by a specific ECDb version or higher.">

```

Comment:
Are these two custom attributes orthogonal? i.e., are there scenarios in which it's ok to import but not to use, or vice versa; or scenarios in which import requires a different profile version than use?

Reply:
Yes that's the idea. Some concepts can be imported by old runtimes, while others require additional logic during import.
Similarly if a new database imported something and ran the additional logic, it may be fine to use it in an older database.